app/Http/Controllers/Order/RenewController.php

```diff
@@ -69,9 +69,14 @@ public function renewBySubId($id, $planid, $payment_method, $cost, $code)
             $sub->update_ends_at = $updatesExpiry;
             $sub->support_ends_at = $supportExpiry;
             $sub->save();
-            $licenseStatus = StatusSetting::pluck('license_status')->first();
-            if ($licenseStatus == 1) {
-                $this->editDateInAPL($sub, $updatesExpiry, $licenseExpiry, $supportExpiry);
+
+            if (Order::where('id', $sub->order_id)->value('license_mode') == 'File') {
+                Order::where('id', $sub->order_id)->update(['is_downloadable'=> 0]);
+            } else {
+                $licenseStatus = StatusSetting::pluck('license_status')->first();
+                if ($licenseStatus == 1) {
```

Comment:
No need to use comparison operator

Reply:
We need to specify the value it's not just boolean. it has 0,1 and 2 as values here